### ISSUE TITLE
Global arrays not initiating with good values

### ISSUE THREAD

**Issue by azalty** [azalty/sm-no-dupe-account#24]
**Describe the bug**
These global arrays should initiate with -1 for each index, but it only does for index 0:
```sp
int g_iClientDatabasePlaytime[MAXPLAYERS + 1] = -1;
int g_iClientDatabaseSteamLevel[MAXPLAYERS + 1] = -1;
int g_iClientDatabaseSteamAge[MAXPLAYERS + 1] = -1;
int g_iClientDatabaseCSGOLevel[MAXPLAYERS + 1] = -1;
```

My bad, I thought this worked.

**Error logs**
None.

**Bug priority**
Minor. Shouldn't cause too much problems, since when the first players at these indexes disconnects, it will fix the value to -1.
It could cause problems with the first players connecting to the server.

**Fix**
Correct syntax:
```sp
int g_iClientDatabasePlaytime[MAXPLAYERS + 1] = {-1, ...};
int g_iClientDatabaseSteamLevel[MAXPLAYERS + 1] = {-1, ...};
int g_iClientDatabaseSteamAge[MAXPLAYERS + 1] = {-1, ...};
int g_iClientDatabaseCSGOLevel[MAXPLAYERS + 1] = {-1, ...};
```

I'll do a commit soon, but since I'm not very available atm, I prefer to wait a bit.
This issue is here so I don't forget about that problem.

**Other**
There might be other arrays that suffer from this problem. I'll take a look about that.

**Comment by azalty** [comment]
This is fixed in the beta 'main' branch. It'll be fixed in the next stable release too.

**Comment by azalty** [comment]
Fixed in the 1.5.0 stable release.
